packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionContainerlessStartExecutor.ts

```diff
@@ -19,22 +19,27 @@ import {
 import { ForceFunctionStartExecutor } from './ForceFunctionStartExecutor';
 
 export class ForceFunctionContainerlessStartExecutor extends ForceFunctionStartExecutor {
+  private process: LocalRunProcess | undefined | void;
```

Comment:
nit: void should encompass undefined here so I think it could just be `LocalRunProcess | void`

Reply:
Removing void results in error at `this.process = await localRun.exec()`
```
(property) ForceFunctionContainerlessStartExecutor.process: LocalRunProcess | undefined
Type 'void | LocalRunProcess' is not assignable to type 'LocalRunProcess | undefined'.
  Type 'void' is not assignable to type 'LocalRunProcess | undefined'.ts(2322)
```

Removing undefined results in error at `private process: LocalRunProcess | void;`
```
(property) ForceFunctionContainerlessStartExecutor.process: void | LocalRunProcess
Property 'process' has no initializer and is not definitely assigned in the constructor.ts(2564)
```

And looks like void can't be assigned to e.g. `this.process = undefined;` -> `this.process = void;` results in error

Reply:
word.  Thanks for trying.